src/engraving/dom/measure.cpp

```diff
@@ -947,6 +947,7 @@ void Measure::remove(EngravingItem* e)
         if (e->isJump() || (e->isMarker() && toMarker(e)->isRightMarker())) {
             setProperty(Pid::REPEAT_JUMP, false);
         }
+    // FALLTHROUGH
```

Comment:
see https://github.com/musescore/MuseScore/pull/25670

Reply:
You mean better using `[[fallthrough]];` here?
That'd be true for many other places that currently are using `// FALLTHROUGH`.
Or are you just pointing out a possible merge conflict?

Reply:
That PR is hanging in libbo since  4 months now...

Reply:
`// FALLTHROUGH` is the most portable way of fixing this, not every compiler understands `[[fallthrough]];`

Reply:
Actually this one might be needed for 4.5.0 too.